scripts/rebuild-testbench.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2018 Intel Corporation. All rights reserved.
```

Comment:
The copyright needs fixed here to your employers and 2020

Reply:
I am student working under GSoC program. I don't really have an employer. Can you suggest me what copyright would work here? Thanks!

Reply:
@cujomalainey can you comment on the copyright owner for GSoC ?

Reply:
We will ping the mailing list.

Reply:
@ymdatta maintains the copyright unless SOF has something against it (e.g. copyright assignment) and license is usually what ever is typically inherited through the project.